pycbc/distributions/sampling_from_config.py

```diff
@@ -0,0 +1,100 @@
+# Copyright (C) 2021  Shichao Wu
+#
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the
+# Free Software Foundation; either version 3 of the License, or (at your
+# option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; If not, see <http://www.gnu.org/licenses/>.
+
+
+import argparse
+import numpy as np
+from pycbc.workflow import WorkflowConfigParser, configuration
```

Comment:
You may want to use the simpler configparser here that won't pull in workflow dependencies. 

https://github.com/gwastro/pycbc/blob/master/pycbc/types/config.py#L53